infrastructure/webdriver/tests/test_load_file.py

```diff
@@ -1,2 +1,2 @@
-def test_load(session, inline):
+def test_load(inline):
```

Comment:
Can you describe this change in the PR description? Was it necessary to pass CI, and what was going wrong here?

Reply:
I believe this was a remnant of trying to solve some of the issues I was seeing with the CI. I remember feeling confident that this was a correct change that was not fixed based on how this test is called (the session argument is no longer passed to this function), but I did not document this and it's been long enough that the exact justification for it is slipping my mind. I've removed it and it seems to not affect the CI, so let's just forget about this change for now 🙂